game/plugin/skills/runecrafting/src/runecrafting.plugin.kts

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
The braces throughout this file need spaces between the brace and enclosed class name.
```
+on_player_event {LoginEvent::class}
```

```
+on_player_event { LoginEvent::class }
```